### PR TITLE
add TV controls channel down and up

### PR DESCRIPTION
**Pull request by Gustry**
add TV controls channel down and up

I tried to add the channel number, but no luck for now. I'm not sure how to give a parameter in my request. I have another look later maybe.

### CODE REVIEW

**Comment by supersaiyanmode** [comment]
Hi @Gustry can you please explain, what you mean by: "I'm not sure how to give a parameter in my request" ? May be with an example? Do you want to do: `tv_control.set_channel(12)` ?

**Comment by supersaiyanmode** [comment]
Ohh, please look at `MediaControl(..).set_volume(..)`:

    "set_volume": {
        "uri": "ssap://audio/setVolume",
        "args": [int],
        "payload": {"volume": arguments(0)}
    },

Think of `"payload"` field as a template, which gets substituted by arguments. In the above code, `argument(0)` retrieves the first argument (=int_val1)that was passed when `set_volume(int_val1)` was called. After substitution the object is sent the the URI in the outer object.

In other words:

    "custom_function": {
        "uri": "ssap://some/uri",
        "args": [int, str],
        "payload": {
            "field1": "blah",
            "field2": [argument(0), argument[1]],
            "field3": argument(1)
        }
    },

upon invocation like: `obj.custom_function(3, "hello")` will produce a payload of `{"field1": "blah", "field2": [3, "hello"], "field3": "hello"}`.

**Comment by supersaiyanmode** [comment]
Thanks @Gustry for this PR! :+1:

**Comment by dagavi** [comment]
In the project referenced in the REAME we can see this: https://github.com/klattimer/LGWebOSRemote/blob/master/LGTV/remote.py#L207

They have:
```
    def inputChannelUp(self, callback=None):
        self.__send_command("request", "ssap://tv/channelUp", None, callback)

    def inputChannelDown(self, callback=None):
        self.__send_command("request", "ssap://tv/channelDown", None, callback)

    def setTVChannel(self, channel, callback=None):
        self.__send_command("request", "ssap://tv/openChannel", {"channelId": channel}, callback)

    def getTVChannel(self, callback=None):
        self.__send_command("request", "ssap://tv/getCurrentChannel", None, callback, "channels")

```

This merge implements channelUp and Down, so I supose that add setChannel and getChannel will be, with your explanition:

```
COMMANDS = {
    "channel_down": {"uri": "ssap://tv/channelDown"},
    "channel_up": {"uri": "ssap://tv/channelUp"},
    "set_channel": {
        "uri": "ssap://tv/openChannel",
        "args": [int],
        "payload": {
            "channelId": argument(0)
        }
    },
    # Based on get_volume
    "get_current_channel": {
        "uri": "ssap://tv/getCurrentChannel",
        "validation": standard_validation,
        "subscription": True,
    }
 }
```

**Comment by supersaiyanmode** [comment]
Hi @dagavi, thanks so much! The new APIs around TV Channels sound good to me. Would you be open to creating a PR with only those commands that work (in other words minus the channel number)?
